EventAggregator/EventAggregator.cs

```diff
@@ -106,6 +135,15 @@ public interface IEventPublisher
          System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1026:DefaultParametersShouldNotBeUsed")]
         void SendMessage<TMessage>(Action<Action> marshal = null)
             where TMessage : new();
+#if ASYNC
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1026:DefaultParametersShouldNotBeUsed")]
+        Task SendMessageAsync<TMessage>(TMessage message, Func<Func<Task>, Task> marshal = null);

```

Comment:
I just wanted to keep the same functionality as in the sync method. If you want I can remove the marshal parameter for the async method.